tools/docker/image-build.sh

```diff
@@ -59,16 +59,18 @@ main() {
 
     for image in "${build_images[@]}"; do
         image_fixed="$(printf "%s" "$image" | tr -cs 'A-Za-z0-9_' '-')"
-        full_image="${DOCKER_REGISTRY}prplmesh-$image_fixed$postfix$TAG"
-        info "Generating $image docker image ($full_image)"
-        run docker image build \
-            --tag "$full_image" \
-            "${scriptdir}/${image}" || exit $?
-        info "Generated $full_image"
+        for tag in "${tags[@]}"; do
+            full_image="${DOCKER_REGISTRY}prplmesh-$image_fixed$postfix$tag"
+            info "Generating $image docker image ($full_image)"
+            run docker image build \
+                --tag "$full_image" \
+                "${scriptdir}/${image}" || exit $?
```

Comment:
It's a bit weird to do the full docker build again when there are multiple tags, but since it's going to use the cache anyway, I guess it's OK.